### ISSUE TITLE
The system extension required for mounting FUSE volumes could not be loaded

### ISSUE THREAD

**Issue by Cubytus** [osxfuse/osxfuse#853]
Hi there, 

after upgrading to Mojave (No Catalina for now as I still have un-replaced 32 bits apps, and I feel that a new OS every year is too fast), I wanted to check a VeraCrypt volume. VC starts, but can't mount encrypted volumes. So I proceeded on reinstalling macFUSE 4.2.1, and found the following error:

> The system extension required for mounting FUSE volumes could not be loaded.
> 
> Please open the Security & Privacy System Preferences pane, got to the General preferences and allow loading system software from developper "Benjamin Fleischer" […]

Which I did, only to find out that I already allowed installation of software from any source (I still use small freeware from independent developers).

Any insight?

**Comment by bfleischer** [comment]
> Which I did, only to find out that I already allowed installation of software from any source (I still use small freeware from independent developers).

The setting you are referring to is not the same as allowing macOS to load a specific kernel extension from a third-party developer. You should see an "Allow" button on the Security & Privacy System Preferences pane when you are promoted by the macFUSE installer to go to system preferences. 

If you are not seeing the "Allow" button, then you might be using some third-party software that is capable of emulating keyboard or mouse events. In this case macOS does not show the "Allow" button. This is a macOS security feature to ensure  that it's actually the user who is giving consent by clicking on the "Allow" button, instead of a software automating the process.

**Comment by Cubytus** [comment]
Interesting. In fact, I DID have two software that could match this description: Karabiner, needed for mapping a non-Apple keyboard, and AnyDesk. I uninstalled them both for the sake of this test, tried to reinstall macFUSE 4.2.3, to no avail. Still no "Allow" button.

Is Mac OS X 10.14 known to consider these software as potentially emulating keyboard or mouse? Password managers, maybe? That's pretty vague and wide-ranging…

**Comment by bfleischer** [comment]
Karabiner is known to cause issues with the Allow button. 

Similar issues have been raised a few times before. Please see https://github.com/osxfuse/osxfuse/issues/654, https://github.com/osxfuse/osxfuse/issues/437, https://github.com/osxfuse/osxfuse/issues/667, https://github.com/osxfuse/osxfuse/issues/783

**Comment by Cubytus** [comment]
The problematic software-hardware you posted in #654 boils down to "practically anything can trigger this undesirable behaviour. I mean, although I don't, many people have Chrome. I also have VLC, Screen Sharing activated, and never thought of these as generating "input events", much less as a security risk.

I tried the solution pointed out in [https://github.com/osxfuse/osxfuse/issues/783#issuecomment-781568532](url) , and was able to mount a VeraCrypt volume, but still no "Allow" button.
![Captura de Pantalla 2021-11-27 a la(s) 20 40 59 ](https://user-images.githubusercontent.com/5182287/143726644-99b6e502-921d-499f-a847-c44b43d90e9d.png)

I then reinstalled Karabiner and got the same warning about allowing a kernel extension, which *did* appear in System Preferences:

![Captura de Pantalla 2021-11-27 a la(s) 21 35 57 ](https://user-images.githubusercontent.com/5182287/143726755-ea862ee0-319d-421e-8e30-b3c71085f02a.png)
Not sure where, in the combination uninstall-reboot-reinstall, the issue disappeared.

So, even if I don't know exactly how, problem solved.
